### PR TITLE
workflow:feat - Automating the release notes

### PR DESCRIPTION
**Pull request by wiliansilvazup**
In this PR I'll update Release Beta, RC and Final to set automatic
body in release notes. I changed file release-new.yml to
release-final.yml and I changed file alpha.yml to release-alpha.yml

Signed-off-by: Wilian Gabriel <wilian.silva@zup.com.br>

### CODE REVIEW

File: .github/workflows/release-alpha.yml [hunk 1]
```diff
@@ -84,12 +84,12 @@ jobs:
           push: true
           context: .
           file: ./deployments/Dockerfile
-          tags: horuszup/horusec-cli:alpha
+          tags: wiliansilvazup/horusec-cli:alpha
```
Comment (nathanmartinszup) [inline .github/workflows/release-alpha.yml:87]: I believe you forgot to change the docker hub repository.

Comment (wiliansilvazup) [reply .github/workflows/release-alpha.yml:87]: oh yeah tnks

**Comment by wiliansilvazup** [comment]
This PR Depend of the https://github.com/ZupIT/horusec-devkit/pull/166

**Comment by wiliansilvazup** [comment]
I created a validation to check if pipeline failed and remove tag created on github: https://github.com/wiliansilvazup/horusec/runs/5280186446?check_suite_focus=true

And now is possible see the users on Release Notes :partying_face: 
- Action: https://github.com/wiliansilvazup/horusec/runs/5280295946?check_suite_focus=true
- Release Notes: 
image
image

**Comment by matheusalcantarazup** [comment]
Wouldn't be better change the commit message to `workflow:feat - Automating the release notes`?

**Comment by wiliansilvazup** [comment]
> Wouldn't be better change the commit message to `workflow:feat - Automating the release notes`?

yeah, I'll adjust this.

File: go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/ZupIT/horusec
 go 1.17
 
 require (
-	github.com/ZupIT/horusec-devkit v1.0.22
+	github.com/ZupIT/horusec-devkit v1.0.23-0.20220221194704-5b84b7847e09
```
Comment (nathanmartinszup) [inline go.mod:6]: A new devkit release should be released before merge

Comment (wiliansilvazup) [reply go.mod]: done

**Comment by wiliansilvazup** [comment]
Wait the PR https://github.com/ZupIT/horusec/pull/996 to fix problems on sintax

**Comment by matheusalcantarazup** [comment]
@wiliansilvazup resolve the conflicts  please

**Review by nathanmartinszup — APPROVED** [review]

**Pull request merged**